packages/patterns-react/src/patterns/CardSection/index.js

```diff
@@ -5,4 +5,4 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-export { default as CardsWithoutImages } from './CardsWithoutImages';
+export { default as CardSection } from './CardSection';
```

Comment:
@mkothur `SimpleCards` and `ImageCards` should probably be exported here too.

Reply:
Okay